migrations/main.sql

```diff
@@ -264,7 +264,7 @@ CREATE TABLE IF NOT EXISTS user_request_review(
     room_number integer,
     start_date_accommodation timestamp,
     end_date_accommodation timestamp,
-    total_sum float,
+    total_sum decimal,
```

Comment:
i think we need to define limit of digits after com. So please add to decimal(5, 2)